Source/WebKit/UIProcess/ios/ViewGestureControllerIOS.mm

```diff
@@ -376,7 +376,7 @@ - (BOOL)isNavigationSwipeGestureRecognizer:(UIGestureRecognizer *)recognizer
             if (auto gestureController = controllerForGesture(pageID, gestureID))
                 gestureController->willCommitPostSwipeTransitionLayerTree(true);
         });
-        drawingArea->hideContentUntilPendingUpdate();
+        drawingArea->hideContentUntilPendingUpdate(WebProcessProxy::ShouldStartProcessThrottlerActivity::Yes);
```

Comment:
I think you could get away with this being a No as well. It's the status quo before this series of patches, it's only relevant if you bg the app after swiping, and presumably would just hold the process awake needlessly if you were to do so.